libs/application/templates/p-sign/src/forms/application.ts

```diff
@@ -0,0 +1,389 @@
+                    .nationalRegistry.data as User
+                  return (
+                    nationalRegistry.address.postalCode +
+                    ', ' +
+                    nationalRegistry.address.city
+                  )
+                },
+              }),
+              buildTextField({
+                id: 'email',
+                title: m.applicantsEmail,
+                variant: 'email',
+                width: 'half',
+                backgroundColor: 'blue',
+                defaultValue: (application: Application) => {
+                  const data = application.externalData.userProfile
+                    .data as UserProfile
+                  return data.email
+                },
+              }),
+              buildTextField({
+                id: 'phone',
+                title: m.applicantsPhoneNumber,
+                variant: 'tel',
+                width: 'half',
+                backgroundColor: 'blue',
+                defaultValue: (application: Application) => {
+                  const data = application.externalData.userProfile
+                    .data as UserProfile
+                  return data.mobilePhoneNumber
+                },
+              }),
+              buildDateField({
+                id: 'validityPeriod',
+                title: m.cardValidityPeriod,
+                width: 'half',
+                backgroundColor: 'white',
+                defaultValue: () => new Date().toISOString(),
+              }),
+            ],
+          }),
+        ],
+      }),
+      buildSection({
+        id: 'photo',
+        title: m.qualityPhotoSectionTitle,
+        children: [
+          buildMultiField({
+            id: 'info',
+            title: m.qualityPhotoTitle,
+            condition: (_, externalData) => {
+              return (
+                (externalData.qualityPhoto as QualityPhotoData)?.data
+                  ?.success === true
+              )
+            },
+            children: [
+              buildDescriptionField({
+                id: 'descriptionPhoto',
+                title: '',
+                description: m.qualityPhotoExistingPhotoText,
+              }),
+              buildCustomField({
+                title: '',
+                id: 'qphoto',
+                component: 'QualityPhoto',
+              }),
+              buildRadioField({
+                id: 'qualityPhoto',
+                title: '',
+                width: 'half',
+                disabled: false,
+                options: [
+                  { value: 'yes', label: m.qualityPhotoUseExistingPhoto },
+                  { value: 'no', label: m.qualityPhotoUploadNewPhoto },
+                ],
+              }),
+              buildCustomField({
+                id: 'photoAttachment',
+                title: '',
+                component: 'PhotoUpload',
+                condition: (answers: FormValue) =>
+                  answers.qualityPhoto === 'no',
```

Comment:
I see several conditions where a variable is checked against `'yes'` or `'no'`. This comparison should use the constants `YES` and `NO` from the `constants.ts` file.